src/components/Layout/Section/RSVDetails.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react';
+import { ButtonLink } from '~/components/Links/ButtonLink';
+import { BMDVFunding } from '~/components/Layout/BMDVFunding';
+import Map from 'react-map-gl';
+import maplibregl, { LngLatBoundsLike } from 'maplibre-gl';
+import 'maplibre-gl/dist/maplibre-gl.css';
+import { RSVSegment } from './RSVSegment';
+import { RSVPopup } from './RSVPopup';
+
+type Props = {
+  meta: {
+    finished: string;
+    state: string;
+    cost: string;
+    general: {
+      name: string;
+      description: string;
+      to: string;
+      from: string;
+      ref: string;
+    };
+    references: {
+      website: string;
+    };
+  };
+  geometry: {
+    bbox: [number, number, number, number];
+    features: Array<GeoJSON.Feature>;
+  };
+};
+
+const mapBounds: LngLatBoundsLike = [
+  4.98865807458, 47.3024876979, 16.0169958839, 54.983104153,
+];
+
+export const RSVDetails: React.VFC<Props> = ({ meta, geometry }) => {
+  const [info, setInfo] = useState({
+    lng: 0,
+    lat: 0,
+    properties: null,
+  });
+  const [selected, setSelected] = useState(undefined);
+
+  const updateInfo = (event) => {
+    const { lng, lat } = event.lngLat;
+    const [feature] = event.features;
+    setInfo({
+      lng,
+      lat,
+      properties: feature.properties,
+    });
+    setSelected(feature.properties.id);
+  };
+
+  return (
+    <div className="relative bg-white">
+      <div className="h-56 overflow-hidden bg-emerald-300 shadow-xl sm:h-72 lg:absolute lg:left-0 lg:h-full lg:w-1/2 lg:rounded-br-2xl">
+        <Map
+          initialViewState={{
+            zoom: 8,
+            bounds: geometry.bbox,
+          }}
+          mapLib={maplibregl}
+          mapStyle="https://basemaps.cartocdn.com/gl/positron-gl-style/style.json"
```

Comment:
TODO: We need to update that to use our own map style.

This should also defined globally in the app, since we may use the map in other components to. See Draft Issue in Board.